### PR TITLE
HBASE-23743 Add HBase 1.4.13 to the downloads page

### PR DESCRIPTION
**Pull request by jatsakthi**

### CODE REVIEW

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 14s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   6m 10s |  master passed  |
| +1 :green_heart: |  mvnsite  |  17m 46s |  master passed  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   5m 37s |  the patch passed  |
| +1 :green_heart: |  mvnsite  |  17m 28s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  2s |  The patch has no ill-formed XML file.  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 20s |  The patch does not generate ASF License warnings.  |
|  |   |  49m 50s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.6 Server=19.03.6 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1226/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1226 |
| Optional Tests | dupname asflicense mvnsite xml |
| uname | Linux 7940c28e99be 4.15.0-74-generic #84-Ubuntu SMP Thu Dec 19 08:06:28 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/Base-PreCommit-GitHub-PR_PR-1226/out/precommit/personality/provided.sh |
| git revision | master / a420f0482e |
| Max. process+thread count | 86 (vs. ulimit of 10000) |
| modules | C: . U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1226/1/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

File: src/site/xdoc/downloads.xml [hunk 1]
```diff
@@ -113,23 +113,23 @@ under the License.
...
-        1.4.12
+        1.4.13
       </td>
       <td style="test-align: left">
-        2019/11/29
+        2020/02/29
       </td>
       <td style="test-align: left">
-        <a href="https://apache.org/dist/hbase/hbase-1.4.12/compat-check-report.html">1.4.11 vs 1.4.12</a>
+        <a href="https://apache.org/dist/hbase/1.4.13/compat-check-report.html">1.4.12 vs 1.4.13</a>
```
Comment (busbey) [inline src/site/xdoc/downloads.xml]: did a release script choose this directory name? I want to get us consistently naming the main repo releases as "hbase-X.Y.Z".

Comment (jatsakthi) [reply src/site/xdoc/downloads.xml]: oops! my bad @busbey . Let me keep this in mind from next time, because I didn't use the script for this one.

**Comment by jatsakthi** [comment]
Thanks for the reviews @busbey , @HorizonNet , @virajjasani  Let me push this one.

**Pull request merged**